CODING_STYLE.md

```diff
@@ -0,0 +1,172 @@
+# Zilliqa Coding Style
```

Comment:
Perhaps we can tag the rules on this page that are automatically checked by our tools during compilation?  So it's easier to know which are enforced by tools and which still need to be followed manually by coders.

Reply:
already have given some indication in CONTRIBUTING.md